estate/__init__.py

```diff
@@ -0,0 +1 @@
+from . import models
```

Comment:
There is a missing empty line at the end of the file. You should configure your IDE to make that automatically. This comment also applies to all files in the PR